bigbluebutton-html5/imports/ui/components/nav-bar/talking-indicator/container.jsx

```diff
@@ -36,12 +37,16 @@ export default withTracker(() => {
         callerName, talking, color, voiceUserId, muted, intId,
       } = usersTalking[i];
 
+      const user = Users.findOne({ userId: voiceUserId });
```

Comment:
```suggestion
      const user = Users.findOne({ userId: voiceUserId }, {fields: {name: 1}});
```

Only obtain the name. Minor thing

Reply:
Sure. Well pointed